guides/v2.3/install-gde/prereq/install-rabbitmq.md

```diff
@@ -101,13 +102,31 @@ where:
 |`--amqp-virtualhost`|The virtual host for connecting to RabbitMQ. The default is `/`.
 |`--amqp-ssl`|Indicates whether to connect to RabbitMQ. The default is `false`. If you set the value to true, see Configure SSL for more information.|
 
+## Connect RabbitMQ to {{site.data.var.ce}} or {{site.data.var.ee}} 
+
+If you already had Magento installed and you want to connect it with RabbitMq, add `queue` section in the `<install_directory>/app/etc/env.php` file so that it is similar to the following:
```

Comment:
```suggestion
If you already had Magento installed and you want to connect it to RabbitMQ, add a `queue` section in the `<install_directory>/app/etc/env.php` file so that it is similar to the following:
```